user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementMain.java

```diff
@@ -530,8 +531,37 @@ private void openRegistrationDialog() {
     var registrationUseCase = new MeasurementUpload(asyncService, context,
         ConverterRegistry.converterFor(
             MeasurementRegistrationInformationNGS.class), messageSourceNotificationFactory);
-    DialogBody.with(dialog, registrationUseCase, registrationUseCase);
+    //DialogBody.with(dialog, registrationUseCase, registrationUseCase);
+    var templateComponent = new MeasurementTemplateSelectionComponent(Map.of(Domain.Genomics, new WorkbookDownloadStreamProvider() {
```

Comment:
for multiple entries consider using `Map#ofEntries` instead as it increases readability a lot. 
see https://docs.oracle.com/en/java/javase/24/docs/api/java.base/java/util/Map.html#ofEntries(java.util.Map.Entry...)